### ISSUE TITLE
Demo Down ?

### ISSUE THREAD

**Issue by warrenbuckley** [umbraco/Umbraco.VercelCommerce.Demo#1]
Hiya @mattbrailsford 👋 
Seems like the Demo site is down from clicking the frontend button from this readme and the same link on the [Vercel template page](https://vercel.com/templates/next.js/nextjs-commerce)

It takes us to https://vercel-commerce-demo.umbraco.com/
When trying to add something to the cart/basket we get a vauge error of 

```
Oh no!
There was an issue with our storefront. This could be a temporary issue, please try your action again.
```

![image](https://github.com/umbraco/Umbraco.VercelCommerce.Demo/assets/1389894/4a6bb2f1-4248-4690-99ed-d2d6e244debb)

**Comment by mattbrailsford** [comment]
Yea, I need to look at an alternative resetting approach for this. The demo uses a SQLite database and we reset it nightly, but it is constantly leaving tmp files which messes things up 🤦‍♂️

I think we need an approach that setups up and populates a new SQL Server databsae

**Comment by warrenbuckley** [comment]
Yes in an ideal world, it would be fab if you used SQL Server for this demo, because I am currently trying to create a uMarketingSuite Headless Demo based on top of this with Umbraco Commerce, but uMarketingSuite only works with SQL Server and not SQLite.

Perhaps you could add another project to do a Package Migration Plan in order to do the initial setup of doctypes and content etc that way the SQLite DB is not needed to be checked in and committed ?
